sunpy/cm/tests/test_cm.py

```diff
@@ -0,0 +1,16 @@
+import sunpy.cm
+
+import pytest
+import sunpy.cm as cm
+import matplotlib.pyplot as plt
+
+
+def test_get_cmap():
+    colormap_list = cm.cmlist
+    for cmap in colormap_list.keys():
+        assert colormap_list[cmap] == cm.get_cmap(cmap)
```

Comment:
I think we should test the ```plt.get_cmap``` instead of SunPy.

Reply:
Fixed it.